sunbird-es-utils/src/main/java/org/sunbird/common/factory/EsClientFactory.java

```diff
@@ -0,0 +1,25 @@
+package org.sunbird.common.factory;
+
+import org.sunbird.common.ElasticSearchRestHighImpl;
+import org.sunbird.common.ElasticSearchTcpImpl;
+import org.sunbird.common.inf.ElasticService;
+
+public class EsClientFactory {
+
+  private static ElasticService tcpClient = null;
+  private static ElasticService restClient = null;
+
```

Comment:
Add one method getInstance this method will take String as argument. if value is "tcp" then return tcp client , if value is "rest" then retuen rest client.  and define the method in such a way that caller will get to know what i have to pass.
SO for end user there is only one method.

Reply:
Done,it will reflect in next commit